src/python/pants/backend/python/register.py

```diff
@@ -92,6 +99,16 @@ def register_goals():
   task(name='unpack-wheels', action=UnpackWheels).install()
 
 
+def target_adaptors():
```

Comment:
Given that this is temporary, how would you feel about renaming this to `_target_adaptors`? 

It's not strictly necessary because V2 is unstable so we could remove this without needing a deprecation, but `register.py` is generally very very public so the leading underscore might make it more clear this is experimental.